services/backend/README.md

```diff
@@ -38,7 +38,7 @@ The property users is an array of objects, each one has a *username* and *passwo
 **This property isn't present in the default config file, if you don't provide it in your custom configuration all endpoints related with users will not work**
 
 ### Run
-Run `npm start` to start the server, this will launch nodemon with a delay of 3 seconds, ready to watch your changes.
+Run `yarn start` to start the server, this will launch nodemon with a delay of 3 seconds, ready to watch your changes.
```

Comment:
"Yarn start" should be executed at the root of repository or being at "/services/backend" ? I mean, in the future will be necessary to run each microservice individually or there is a way to run all microservices in the workspace?

Reply:
1. `yarn start` from the root of the project won't work because script `start` is not defined in the root `package.json`
2. `yarn start` from `/services/backend` will start the service as we know it
3. Available scripts to be run in every workspace from the root of the project are: `lint` and `docs`
4. I forgot to create the global readme :woman_facepalming: I'll do that right now

Reply:
if you want to run an specific service from the root of the project, you can `yan workspace backend run start` or doing it from the `/services/backend` folder